lib/pymedphys/_experimental/chartchecks/compare.py

```diff
@@ -185,22 +188,27 @@ def compare_to_mosaiq(dicom_table, mos_table):
                     [mos_table.iloc[field][label]], columns=[label]
                 )
 
-                dicom_list[label] = add_dicom
-                mosaiq_list[label] = add_mosaiq
+                dicom_list = pd.concat([dicom_list, add_dicom], axis=1)
+                mosaiq_list = pd.concat([mosaiq_list, add_mosaiq], axis=1)
+
             # continue if the value is not in Mosaiq
             else:
                 continue
 
         values_table = values_table.append(dicom_list, ignore_index=True)
         values_table = values_table.append(mosaiq_list, ignore_index=True)
 
+        dicom_list = pd.DataFrame()
```

Comment:
The word "list" has a very special meaning in Python, given this isn't a Python list I would change this variable's name.